docs/spec/staking/end_block.md

```diff
@@ -1,68 +1,40 @@
 # End-Block 
 
-## Unbonding Validator Queue
+## Validator Set Changes
 
-For all unbonding validators that have finished their unbonding period, this switches their validator.Status
-from sdk.Unbonding to sdk.Unbonded if they still have any delegation left.  Otherwise, it deletes it from state.
+The Tendermint validator set is be updated by state transitions that run at the
```

Comment:
`is be updated`? Please expand on the reasoning here.

Reply:
Or clarify what Tendermint does.